README.md

```diff
@@ -1 +1,41 @@
-# snips-serverless
\ No newline at end of file
+# snips-serverless 
+
+This VS Code extension adds snippets for Serverless in your editor for typescript.
+
+### TypeScript Snippets (.ts)
+
+| Snippet             | Purpose                                    |
+|---------------------|--------------------------------------------|
+| `create`            | Snippet for Create Endpoint Lambda         |
+| `list`              | Snippet for List Endpoint Lambda           |
+| `get`               | Snippet for Get Endpoint Lambda            |
+| `update`            | Snippet for Update Endpoint Lambda         |
+| `delete`            | Snippet for Delete Endpoint Lambda         |
+| `testcreate`        | Snippet for testing Create Endpoint Lambda |
+| `testlist`          | Snippet for testing List Endpoint Lambda   |
+| `testget`           | Snippet for testing Get Endpoint Lambda    |
+| `testupdate`        | Snippet for testing Update Endpoint Lambda |
+| `testdelete`        | Snippet for testing Delete Endpoint Lambda |
+| `arrow`             | Snippet for Arrow Function                 |
+| `it`                | Snippet for it (testing)                   |
+| `unroll`            | Snippet for unroll (testing)               |
+| `describe`          | Snippet for describe (testing)             |
```

Comment:
These snippets are already added in the `snips-react`. Do we really need them here as well?

Reply:
As the three extensions [`snips-react`, `snips-sequelize` and `snips-serverless`] are meant to be maintained separately, it only makes sense to keep the common snippets in all of them as one working on **react** (using `snips-react`) should not install `snips-sequelize` extension just to get the testing snippets. Leaving this as-is.